protobuf/msgs.proto

```diff
@@ -35,8 +39,14 @@ message Message {
 		CSPaillierOpening cs_paillier_opening = 12;
 		CSPaillierProofData cs_paillier_proof_data = 13;
 		CSPaillierProofRandomData cs_paillier_proof_random_data = 14;
+		PseudonymsysCACertificate pseudonymsys_ca_certificate = 15;
+		PseudonymsysNymGenProofRandomData pseudonymsys_nym_gen_proof_random_data = 16;
+		PseudonymsysIssueProofRandomData pseudonymsys_issue_proof_random_data = 17;
+		DoubleBigInt double_bigint = 18;
+		PseudonymsysTransferCredentialData pseudonymsys_transfer_credential_data = 19;
 	}
-	int32 clientId = 15;
+	int32 clientId = 20;
+	string ErrorMsg = 21;
```

Comment:
Yes, indeed, I didn't realize. Thanks for pointing this out. I agree with this addition, but please rename the field from `ErrorMsg` to `ProtocolError` + recompile with `protoc` and commit an updated `msgs.pb.go`.